hotsos/defs/tests/scenarios/kernel/kernlog_calltrace_hungtasks.yaml

```diff
@@ -0,0 +1,37 @@
+target-name: kernlog_calltrace_hungtasks.yaml
```

Comment:
this should be:

target-name: kernlog_calltrace.yaml

Reply:
thank you!